File: app/src/main/java/com/babestudios/companieshouse/data/model/insolvency/Insolvency.java
package com.babestudios.companieshouse.data.model.insolvency;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class Insolvency {
	@SerializedName("cases")
	public List<InsolvencyCase> cases = new ArrayList<>();
	@SerializedName("etag")
	public String etag;
	@SerializedName("status")
	public List<Object> status = new ArrayList<>();
}

File: app/src/main/java/com/babestudios/companyinfouk/data/model/insolvency/Insolvency.kt
package com.babestudios.companyinfouk.data.model.insolvency

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName

import java.util.ArrayList

@Keep
class Insolvency {
	@SerializedName("cases")
	var cases: List<InsolvencyCase> = ArrayList()
	@SerializedName("etag")
	var etag: String? = null
	@SerializedName("status")
	var status: List<Any> = ArrayList()
}
